pwiz.py

```diff
@@ -74,6 +74,9 @@ def _print_table(table, seen, accum=None):
         if not preserve_order:
             columns = sorted(columns)
         primary_keys = database.primary_keys[table]
+        composite_indices = [index_data for index_data in database.indexes[table] if len(index_data.columns) > 1]
+        composite_unq_cols = [col for idx in composite_indices for col in idx.columns if idx.unique]
+        composite_nonunq_cols = [col for idx in composite_indices for col in idx.columns if not idx.unique]
```

Comment:
These lines look to be > 79 characters. Additionally nested list comprehensions are typically something I prefer to avoid in my code.